### PR TITLE
removed all @SuppressWarnings annotations

### PR DESCRIPTION
**Pull request by msgroi**

### CODE REVIEW

File: src/test/java/com/salesforce/dynamodbv2/DocGeneratorRunner.java [hunk 1]
```diff
@@ -472,16 +472,14 @@ void printToTable(List<String> columnNames, List<Object[]> data) {
             return createItem(hashKeyField, value, "someField", "value-" + value);
         }
 
-        @SuppressWarnings("all")
-        private String buildTableName(String table, int ordinal) {
+            private String buildTableName(String table, int ordinal) {
```
Comment (horwitzj) [inline src/test/java/com/salesforce/dynamodbv2/DocGeneratorRunner.java]: should this have four spaces removed from the beginning of the line?

File: src/main/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepo.java [hunk 2]
```diff
@@ -235,22 +235,19 @@ public MtDynamoDbTableDescriptionRepoBuilder withTableDescriptionTableName(Strin
...
-        public MtDynamoDbTableDescriptionRepoBuilder withTableDescriptionTableDataField(
+            public MtDynamoDbTableDescriptionRepoBuilder withTableDescriptionTableDataField(
             String tableDescriptionTableDataField) {
             this.tableDescriptionTableDataField = tableDescriptionTableDataField;
             return this;
         }
 
-        @SuppressWarnings("unused")
-        public MtDynamoDbTableDescriptionRepoBuilder withDelimiter(String delimiter) {
+            public MtDynamoDbTableDescriptionRepoBuilder withDelimiter(String delimiter) {
```
Comment (horwitzj) [inline src/main/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepo.java]: should this have four spaces removed from the beginning of the line?

File: src/main/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepo.java [hunk 3]
```diff
@@ -235,22 +235,19 @@ public MtDynamoDbTableDescriptionRepoBuilder withTableDescriptionTableName(Strin
...
-        public MtDynamoDbTableDescriptionRepoBuilder withTableDescriptionTableHashKeyField(
+            public MtDynamoDbTableDescriptionRepoBuilder withTableDescriptionTableHashKeyField(
             String tableDescriptionTableHashKeyField) {
             this.tableDescriptionTableHashKeyField = tableDescriptionTableHashKeyField;
             return this;
         }
 
-        @SuppressWarnings("unused")
-        public MtDynamoDbTableDescriptionRepoBuilder withTableDescriptionTableDataField(
+            public MtDynamoDbTableDescriptionRepoBuilder withTableDescriptionTableDataField(
```
Comment (horwitzj) [inline src/main/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepo.java]: should this have four spaces removed from the beginning of the line?

File: src/main/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepo.java [hunk 4]
```diff
@@ -235,22 +235,19 @@ public MtDynamoDbTableDescriptionRepoBuilder withTableDescriptionTableName(Strin
             return this;
         }
 
-        @SuppressWarnings("unused")
-        public MtDynamoDbTableDescriptionRepoBuilder withTableDescriptionTableHashKeyField(
+            public MtDynamoDbTableDescriptionRepoBuilder withTableDescriptionTableHashKeyField(
```
Comment (horwitzj) [inline src/main/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepo.java]: should this have four spaces removed from the beginning of the line?

File: src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilder.java [hunk 5]
```diff
@@ -213,8 +210,7 @@ private void addStreamSpecification(CreateTableRequestBuilder createTableRequest
 
     private void addSi(CreateTableRequestBuilder createTableRequestBuilder,
                        DynamoSecondaryIndexType indexType,
-                       @SuppressWarnings("all")
-                           ScalarAttributeType hashKeyType,
+                                              ScalarAttributeType hashKeyType,
```
Comment (horwitzj) [inline src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilder.java]: one last seemingly over-spaced line

Comment (msgroi) [reply src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilder.java]: Fixed.

**Pull request merged**